### ISSUE TITLE
Random python children are spawned and not killed.

### ISSUE THREAD

**Issue by KarlWithK** [pappasam/coc-jedi#36]
Hello, so recently I was trying out coc-jedi and it works after following the instructions. However, I found that after closing vim I would be left with two python processes running in the background using a considerable amount of CPU usage.
![image](https://user-images.githubusercontent.com/31679231/116308097-19c0a580-a775-11eb-8524-9a7bb42c376c.png)

I tested this, and I believe this plugin is the reason why it behaves like this. How would I fix this? Here is the output from CocInfo:
```log
## versions

vim version: NVIM v0.5.0-dev+1282-gfbe18d9ca
node version: v16.0.0
coc.nvim version: 0.0.80-bf98939a8c
coc.nvim directory: /home/karl/.vim/plugged/coc.nvim
term: xterm-256color
platform: linux

## Log of coc.nvim

2021-04-27T16:41:18.607 INFO (pid:287025) [extensions] - Skipped load vim plugin from "/home/karl/.vim/plugged/coc-jedi", "coc-jedi" already global extension.
2021-04-27T16:41:18.768 INFO (pid:287025) [coc-discord-rpc] - === Extension activated ===
2021-04-27T16:41:18.768 INFO (pid:287025) [coc-discord-rpc] - Extension Name: leonardssh.coc-discord-rpc.
2021-04-27T16:41:18.768 INFO (pid:287025) [coc-discord-rpc] - Extension Version: 4.2.1-6f0854b2.
2021-04-27T16:41:19.627 INFO (pid:287025) [coc-discord-rpc] - Connected to Discord Gateway!
2021-04-27T16:41:20.546 INFO (pid:287025) [services] - registered service "jedi"
2021-04-27T16:41:20.547 INFO (pid:287025) [services] - jedi-language-server state change: stopped => starting
2021-04-27T16:41:20.780 INFO (pid:287025) [language-client-index] - Language server "jedi" started with 287044
2021-04-27T16:41:20.856 INFO (pid:287025) [plugin] - coc.nvim 0.0.80-bf98939a8c initialized with node: v16.0.0 after 2341ms
2021-04-27T16:41:21.487 INFO (pid:287025) [services] - jedi-language-server state change: starting => running
2021-04-27T16:41:21.497 INFO (pid:287025) [services] - service jedi started
2021-04-27T16:41:24.684 INFO (pid:287025) [attach] - receive notification: showInfo []

```

Thank you.

**Comment by KarlWithK** [comment]
Okay, so I looked through my config and I realized that did two things that might have been my fault.
1) I installed both this plugin though coci and the plug way. Perhaps that created conflict.
2) After I removed this line the processes did not appear: `  " autocmd VimLeavePre * :call coc#rpc#kill()` But i'm a bit confused why.

Now, the reason I had this was because sometimes I accidentally killed my terminal and this would help me kill coc.  Without this some process will left for hang if I do not kill vim properly. I'm wondering if there is a solution to using this and jedi.
